server/resources.py

```diff
@@ -22,5 +23,28 @@ def get(self, lecture_id):
             'comments': comments
         }
 
+    def post(self, lecture_id):
+        db_lectures = Lecture.query.filter(Lecture.id == lecture_id).all()
+
+        if not db_lectures:
+            abort(404, message="Lecture {} does not exist".format(lecture_id))
+
+        parser = reqparse.RequestParser()
+        parser.add_argument('data', help='Text content of comment')
```

Comment:
I don't really have any rule for that. This is just me being inconsistent... Is that how you do it?